bin/sqlline.py

```diff
@@ -123,8 +126,11 @@ def kill_child():
     '" -Dlog4j2.configurationFile=file:' + os.path.join(phoenix_utils.current_dir, "log4j2.properties") + \
     disable_jna + \
     " sqlline.SqlLine -d org.apache.phoenix.jdbc.PhoenixDriver" + \
-    " -u jdbc:phoenix:" + phoenix_utils.shell_quote([zookeeper]) + \
-    " -n none -p none --color=" + colorSetting + " --fastConnect=" + tryDecode(args.fastconnect) + \
-    " --verbose=" + tryDecode(args.verbose) + " --incremental=false --isolation=TRANSACTION_READ_COMMITTED " + sqlfile
+    (not args.noconnect and " -u jdbc:phoenix:" + phoenix_utils.shell_quote([zookeeper]) or "") + \
```

Comment:
Most ppl working on this code are not Python experts.
Please avoid this very opaque "bool and string or string2" expression, and use something that Java developers can understand.

Reply:
Python doesn't have a ternary operator and this is the closest thing from the convenience point of view. In other words, this would be equivalent to (in java): `(! args.noconnect ? " -u jdbc:phoenix:" + phoenix_utils.shell_quote([zookeeper]) : "")`. This is not a Python expert level feature and there is nothing that is purposefully made to look cryptic.